packages/bpk-component-datatable/README.md

```diff
@@ -112,7 +114,8 @@ export default () => (
 | Property               | PropType                    | Required | Default Value        |
 | ---------------------- | --------------------------- | -------- | -------------------- |
 | rows                   | arrayOf(Object)             | true     | -                    |
-| children               | arrayOf(BpkDataTableColumn) | true     | -                    |
+| ~~children~~ Replace with `columns` | arrayOf(~~BpkDataTableColumn~~ Object) | true     | -                    |
+| columns                | arrayOf(Object)             | false    | undefined            |
```

Comment:
Should we update the example usage above to use the new property instead of still showing `children` as the method to create columns?

Reply:
oh yes, definitely! I missed this 😅